src/filename_inspector.py

```diff
@@ -0,0 +1,106 @@
+import fnmatch
+import glob
+import os
+import json
+import csv
+
```

Comment:
```suggestion
import logging

logger = logging.getLogger(__name__)
```

Reply:
Used in commit d08c39dc9dd94c5bf27480b7597a8534e8b5ae1b.